tests/test_data_store_edit.py

```diff
@@ -0,0 +1,133 @@
+from unittest import TestCase
+
+import pytest
+import sqlalchemy
+from sqlalchemy.orm import undefer
+
+from pepys_import.core.store.data_store import DataStore
+
+
+class EditDataTestCase(TestCase):
+    def setUp(self):
+        self.store = DataStore("", "", "", 0, ":memory:", db_type="sqlite")
+        self.store.initialise()
+
+        with self.store.session_scope():
+            self.store.populate_reference()
+            self.store.populate_metadata()
+
+            self.platforms = (
+                self.store.session.query(self.store.db_classes.Platform).options(undefer("*")).all()
+            )
+            self.store.session.expunge_all()
+
+    def test_edit_platform_identifier_single(self):
+        self.store.edit_items([self.platforms[0]], {"identifier": "NEWIDENT"})
+
+        all_platforms = self.store.session.query(self.store.db_classes.Platform).all()
+        assert len((all_platforms)) == 4
+
+        result = (
+            self.store.session.query(self.store.db_classes.Platform)
+            .filter(self.store.db_classes.Platform.identifier == "NEWIDENT")
+            .all()
+        )
+        assert len(result) == 1
+
+        assert result[0].name == self.platforms[0].name
+        assert result[0].nationality_name == self.platforms[0].nationality_name
```

Comment:
This test and test below only show that name and nationality_name fields didn't change. I think it would be better to assert that the identifier was changed to "NEWIDENT".

Reply:
The change to NEWIDENT was implicitly tested by the fact that the query was filtering on the identifier, so wouldn't have returned any results if the identifier hadn't been updated properly - but I agree it's better to have it explicitly testing, so I've changed it.